### PR TITLE
241 Hide simple objects for standalone mode

### PR DESCRIPTION
**Pull request by ensemenova**
#241

### CODE REVIEW

File: packages/ketcher-react/src/script/ui/app/toolbar.jsx [hunk 1]
```diff
@@ -224,6 +211,23 @@ function TemplatesList({ active, onAction }) {
   )
 }
 
+function initToolbar() {
+  console.log(global.ketcher)
+  let toolboxItems = [...toolbox]
```
Comment (TomasLight) [inline packages/ketcher-react/src/script/ui/app/toolbar.jsx:216]: it is `const`

File: packages/ketcher-react/src/infrastructure/services/struct/remoteStructServiceProvider.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import IndigoService from './remoteStructService'
+
+class RemoteStructServiceProvider implements StructServiceProvider {
+  private baseUrl!: string
+
+  constructor(baseUrl: string) {
+    this.baseUrl = !baseUrl || /\/$/.test(baseUrl) ? baseUrl : baseUrl + '/';
+  }
+
+  initStructService(options: any): StructService {
```
Comment (AndreiMazol) [inline packages/ketcher-react/src/infrastructure/services/struct/remoteStructServiceProvider.ts:11]: please rename this method to 'createStructService'

File: packages/ketcher-standalone/src/infrastructure/services/struct/standaloneStructServiceProvider.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { StructService, StructServiceProvider } from './structService.types'
+import StandaloneStructService from './standaloneStructService'
+
+class StandaloneStructServiceProvider implements StructServiceProvider {
+  // @ts-ignore
+  initStructService(baseUrl: string, options: any): StructService {
+    ;(global as any).ketcher.standalone = true
```
Comment (TomasLight) [inline packages/ketcher-standalone/src/infrastructure/services/struct/standaloneStructServiceProvider.ts:6]: probably unnecessary semicolon

File: packages/ketcher-react/src/infrastructure/services/struct/remoteStructServiceProvider.ts [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+    ;(global as any).ketcher.apiPath = this.baseUrl
```
Comment (AndreiMazol) [inline packages/ketcher-react/src/infrastructure/services/struct/remoteStructServiceProvider.ts:13]: It should not be a part of StructServiceProvider

File: packages/ketcher-react/src/script/index.js [hunk 5]
```diff
@@ -74,19 +73,14 @@ function showMolfile(clientArea, molString, options) {
...
-  return service
-}
-
 // TODO: replace window.onload with something like <https://github.com/ded/domready>
 // to start early
-export default function init(el, staticResourcesUrl, apiPath, structServiceFn) {
-  ketcher.apiPath = apiPath
-  const params = new URLSearchParams(document.location.search)
-  const createStructServiceFn = structServiceFn || createDefaultStructService
-  if (params.has('api_path')) ketcher.apiPath = params.get('api_path')
```
Comment (AndreiMazol) [inline packages/ketcher-react/src/script/index.js:88]: Please do not remove api_path initialization which is based on query params

File: packages/ketcher-standalone/src/infrastructure/services/struct/standaloneStructServiceProvider.ts [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import { StructService, StructServiceProvider } from './structService.types'
+import StandaloneStructService from './standaloneStructService'
+
+class StandaloneStructServiceProvider implements StructServiceProvider {
+  initStructService(options: any): StructService {
+    ;(global as any).ketcher.standalone = true
```
Comment (AndreiMazol) [inline packages/ketcher-standalone/src/infrastructure/services/struct/standaloneStructServiceProvider.ts:6]: It should be executed once. Please move it into constructor

**Review by AndreiMazol — CHANGES_REQUESTED** [review]

File: packages/ketcher-react/src/infrastructure/services/struct/remoteStructServiceProvider.ts (hunk 2 above)
Comment (ensemenova) [reply packages/ketcher-react/src/infrastructure/services/struct/remoteStructServiceProvider.ts]: refactored

File: packages/ketcher-react/src/script/index.js (hunk 5 above)
Comment (ensemenova) [reply packages/ketcher-react/src/script/index.js:88]: reverted back

File: packages/ketcher-react/src/infrastructure/services/struct/remoteStructServiceProvider.ts (hunk 4 above)
Comment (ensemenova) [reply packages/ketcher-react/src/infrastructure/services/struct/remoteStructServiceProvider.ts]: reverted back

File: packages/ketcher-standalone/src/infrastructure/services/struct/standaloneStructServiceProvider.ts (hunk 6 above)
Comment (ensemenova) [reply packages/ketcher-standalone/src/infrastructure/services/struct/standaloneStructServiceProvider.ts:7]: done

File: packages/ketcher-react/src/script/ui/app/toolbar.jsx (hunk 1 above)
Comment (ensemenova) [reply packages/ketcher-react/src/script/ui/app/toolbar.jsx]: fixed

**Review by TomasLight — APPROVED** [review]

**Review by AndreiMazol — APPROVED** [review]

**Pull request merged**